### PR TITLE
v12patch1: Make master docker volume ephemeral

### PR DESCRIPTION
**Pull request by r7vme**
Backport of https://github.com/giantswarm/aws-operator/pull/1043

Fixes: https://github.com/giantswarm/giantswarm/issues/3655

### CODE REVIEW

File: service/controller/v12patch1/version_bundle.go [hunk 1]
```diff
@@ -74,6 +79,6 @@ func VersionBundle() versionbundle.Bundle {
 			},
 		},
 		Name:    "aws-operator",
-		Version: "3.1.2",
+		Version: "3.1.3",
```
[the remark this answers is not in the document]
Comment (r7vme) [reply service/controller/v12patch1/version_bundle.go]: v13 should be `3.2.0` yes. Patch versions only for patch releases imo.

[the remark this answers is not in the document]
Comment (fgimenez) [reply service/controller/v12patch1/version_bundle.go]: :+1: `3.2.0` makes sense to me.

[the remark this answers is not in the document]
Comment (rossf7) [reply service/controller/v12patch1/version_bundle.go]: Cool I'll update v13 to fix that. I agree patch should just be for patch releases. I messed this up when I added v13.

**Comment by r7vme** [comment]
NOTE: e2e tests passed [here](https://circleci.com/workflow-run/68ef4dad-5563-4df9-b071-9a6e8174fb3f?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-workflow-link). I'll rebase to latest master and merge when build job will be green.

**Pull request merged**